services/gateway/sources/version3/mappings/tokenSummary.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	escrowedAmounts: ['escrowedAmounts', {
+		escrowChainID: '=,string',
+		tokenID: '=,string',
+		amount: '=,string',
+		name: '=,string',
+		symbol: '=,string',
+	}],
+	supportedTokens: ['supportedTokens', {
+		tokenID: '=,string',
+		name: '=,string',
+		symbol: '=,string',
+	}],
+	totalSupply: ['totalSupply', {
+		tokenID: '=,string',
+		amount: '=,string',
+		name: '=,string',
+		symbol: '=,string',
+
+	}],
+};
```

Comment:
Please do not complicate the code. Let us please keep it simple and consistent across the codebase.
```suggestion
const escrowedAmount = {
	escrowChainID: '=,string',
	tokenID: '=,string',
	amount: '=,string',
	name: '=,string',
	symbol: '=,string',
};

const supportedToken = {
	tokenID: '=,string',
	name: '=,string',
	symbol: '=,string',
};

const totalSupplyByToken = {
	tokenID: '=,string',
	amount: '=,string',
	name: '=,string',
	symbol: '=,string',

};

module.exports = {
	escrowedAmount,
	supportedToken,
	totalSupplyByToken,
};

```

Reply:
applied.